apis/compute/v1beta1/computefirewallpolicyrule_reference.go

```diff
@@ -34,16 +34,14 @@ var _ refsv1beta1.ExternalNormalizer = &ComputeFirewallPolicyRuleRef{}
 // holds the GCP identifier for the KRM object.
 type ComputeFirewallPolicyRuleRef struct {
 	// A reference to an externally managed ComputeFirewallPolicyRule resource.
-	// Should be in the format "locations/global/firewallPolicies/<firewallPolicy>/rules/<priority>".
+	// Should be in the format "locations/global/firewallPolicies/{{firewallPolicy}}/rules/{{priority}}".
 	External string `json:"external,omitempty"`
 
 	// The name of a ComputeFirewallPolicyRule resource.
 	Name string `json:"name,omitempty"`
 
 	// The namespace of a ComputeFirewallPolicyRule resource.
 	Namespace string `json:"namespace,omitempty"`
-
-	parent *ComputeFirewallPolicyRuleParent
```

Comment:
See https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3316#discussion_r1870474578 for details